homeassistant/components/zwave_js/api.py

```diff
@@ -408,6 +406,58 @@ def forward_stage(event: dict) -> None:
     connection.send_result(msg[ID], result)
 
 
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/refresh_node_values",
+        vol.Required(ENTRY_ID): str,
+        vol.Required(NODE_ID): int,
+    },
+)
+@async_get_entry
+async def websocket_refresh_node_values(
+    hass: HomeAssistant,
+    connection: ActiveConnection,
+    msg: dict,
+    entry: ConfigEntry,
+    client: Client,
+) -> None:
+    """Refresh node values."""
+    node_id = msg[NODE_ID]
+    node = client.driver.controller.nodes[node_id]
+
+    await node.async_refresh_values()
+    connection.send_result(msg[ID])
+
+
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/refresh_node_cc_values",
+        vol.Required(ENTRY_ID): str,
+        vol.Required(NODE_ID): int,
+        vol.Required(COMMAND_CLASS_ID): int,
```

Comment:
Should we validate that the id is in the command class enum?

Reply:
done